pkg/artifactory/resource_artifactory_group_test.go

```diff
@@ -71,14 +127,13 @@ func testAccCheckGroupDestroy(id string) func(*terraform.State) error {
 		}
 
 		_, resp, err := client.V1.Security.GetGroup(context.Background(), rs.Primary.ID)
+		if resp.StatusCode == http.StatusNotFound {
```

Comment:
If you don't initially check `err`, then you have no guarantee that `resp` isn't`nil` - So, please check that `resp != nil` - I've had this cause me crashes